.github/workflows/tests.yml

```
@@ -266,8 +266,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - {crdb: "latest-v22.2", python: "3.10", impl: "c", libpq: "newest"}
```

Let's spice up the crdb tests by using different psycopg implementations (c and Python), different libpq and Python versions

```suggestion
          - {crdb: "latest-v22.2", python: "3.11", impl: "python", libpq: ""}
```